bower.json

```diff
@@ -28,16 +28,16 @@
     "polymer": "^2.0.0",
     "iron-media-query": "^2.0.0",
     "iron-resizable-behavior": "^2.0.0",
-    "vaadin-control-state-mixin": "vaadin/vaadin-control-state-mixin#^2.1.1",
-    "vaadin-themable-mixin": "vaadin/vaadin-themable-mixin#^1.5.2",
-    "vaadin-text-field": "vaadin/vaadin-text-field#^2.6.0",
-    "vaadin-list-box": "vaadin/vaadin-list-box#^1.3.0",
-    "vaadin-list-mixin": "vaadin/vaadin-list-mixin#^2.4.0",
-    "vaadin-item": "vaadin/vaadin-item#^2.2.0",
-    "vaadin-overlay": "vaadin/vaadin-overlay#^3.4.0",
-    "vaadin-lumo-styles": "vaadin/vaadin-lumo-styles#^1.1.0",
-    "vaadin-material-styles": "vaadin/vaadin-material-styles#^1.1.0",
-    "vaadin-element-mixin": "vaadin/vaadin-element-mixin#^2.3.0"
+    "vaadin-control-state-mixin": "vaadin/vaadin-control-state-mixin#^2.2.1",
+    "vaadin-themable-mixin": "vaadin/vaadin-themable-mixin#^1.6.1",
+    "vaadin-text-field": "vaadin/vaadin-text-field#^2.7.0-alpha1",
+    "vaadin-list-box": "vaadin/vaadin-list-box#^1.4.0-alpha1",
+    "vaadin-list-mixin": "vaadin/vaadin-list-mixin#^2.5.0",
+    "vaadin-item": "vaadin/vaadin-item#^2.3.0-alpha1",
+    "vaadin-overlay": "vaadin/vaadin-overlay#^3.5.0",
+    "vaadin-lumo-styles": "vaadin/vaadin-lumo-styles#^1.6.0",
+    "vaadin-material-styles": "vaadin/vaadin-material-styles#^1.3.2",
```

Comment:
These should have been updated in the previous RTL minor to be aligned with `vaadin-text-field`.